bin/docker-entrypoint

```diff
@@ -52,9 +65,12 @@ help() {
   echo ""
 
   echo "server -- start Redash server (with gunicorn)"
-  echo "worker -- start Celery worker"
-  echo "scheduler -- start Celery worker with a beat (scheduler) process"
-  echo "dev_worker  -- start Celery worker with a beat (scheduler) process which picks up code changes and reloads"
+  echo "celery_worker -- start Celery worker"
+  echo "dev_celery_worker -- start Celery worker process which picks up code changes and reloads"
+  echo "worker -- start a single RQ worker"
+  echo "dev_worker -- start a single RQ worker with code reloading"
+  echo "scheduler -- start an rq-scheduler instance"
+  echo "dev_scheduler -- start an rq-scheduler instance with code reloading"
   echo "celery_healthcheck -- runs a Celery healthcheck. Useful for Docker's HEALTHCHECK mechanism."
```

Comment:
Need to update this...

Reply:
Does it make sense to have an entrypoint for `rq_healthcheck`? Running the command will start a new container, which will obviously not have any workers running within it. The CLI command needs to run within an existing container with a running (or at least registered) worker process.

Reply:
The HEALTHCHECK Docker command uses this entrypoint. It runs it in the context of the same container.

Reply:
👍 76040ae